### PR TITLE
created a test for userprofilepage

### PR DESCRIPTION
**Pull request by dueka**
#### What does this PR do?
Choose one or more:
- Adding tests 
#### Description of Task to be completed? (Copy + Paste Trello card info + link)
Task Description:
Trello Card Link: https://trello.com/c/rIWDpdHi 

#### How has this been tested & how should it be tested in QA?
- The following tests are in place:
    - Test #1  - Certifying the snapshot runs.
- QA should check:
    - Mobile responsiveness
    - Data validation
    - Ex. #3

#### Any background context you want to provide?

- [x] I have performed a self-review of my own code
- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works .
- [x] New and existing unit tests pass locally with my changes

#### Screenshots (if appropriate)
Capture
#### Questions:
- Question #1:
- Question #2:

### CODE REVIEW

File: src/components/templates/__tests__/UserProfileForm.test.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  jestFeatures = render(
+    <Router history={history}>
+      <Provider store={store}>
+        <UserProfileForm />
+      </Provider>
+    </Router>
+  );
+});
+
+describe("Shows all the text nodes on HackathonForm.js that are contained on the making an event form", () => {
```
Comment (AbdelIdir) [inline src/components/templates/__tests__/UserProfileForm.test.js]: This is not accurate, "Shows all the text nodes on HackathonForm.js that are contained on the making an event form"
This is the userprofilepage.js component.
Please modify that description.

Comment (AbdelIdir) [reply src/components/templates/__tests__/UserProfileForm.test.js]: And this is not the making an event form.
It is user profile page

File: src/components/templates/__tests__/UserProfileForm.test.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import React from "react";
```
Comment (AbdelIdir) [inline src/components/templates/__tests__/UserProfileForm.test.js]: why is this file in the same PR ?

Comment (AbdelIdir) [reply src/components/templates/__tests__/UserProfileForm.test.js]: This is the userprofileform.js , but this PR is for userprofilepage.js

File: src/components/templates/__tests__/UserProfileForm.test.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+afterEach(cleanup);
+
+let jestFeatures;
+let mockStore;
+let store;
+
+beforeEach(() => {
+  mockStore = configureStore();
+  // let wrapper;
```
Comment (AbdelIdir) [inline src/components/templates/__tests__/UserProfileForm.test.js]: delete the //let wrapper line

File: src/components/views/__tests__/UserProfilePage.test.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+import { Router, Route } from "react-router";
+import { createMemoryHistory } from "history";
+import { render } from "@testing-library/react";
+import UserProfilePage from "../UserProfilePage";
+const history = createMemoryHistory();
+
+const renderWithRouter = Component =>
+  render(
+    <Router history={history}>
+      <Route component={Component} />
```
Comment (AbdelIdir) [inline src/components/views/__tests__/UserProfilePage.test.js]: This code is not proper/
I think you confused both file names together.
This was the file you was supposed to test on for this PR.

**Comment by dueka** [comment]
[Front-End[Test] /View/UserProfilePage.jsx](https://trello.com/c/rIWDpdHi/221-front-endtest-view-userprofilepagejsx)

**Pull request merged**